starter-core/src/main/java/io/micronaut/starter/build/BuildPlugin.java

```diff
@@ -22,13 +22,15 @@
 import io.micronaut.starter.options.BuildTool;
 import io.micronaut.starter.template.Writable;
 
+import java.util.List;
+
 public interface BuildPlugin extends Ordered {
 
     @NonNull
     BuildTool getBuildTool();
 
     @Nullable
-    Writable getExtension();
+    List<Writable> getExtensions();
```

Comment:
maybe it's easier to just combine the multiple extensions into a single `Writable` instance, could reduce the required changes

Reply:
I reverted this: 

https://github.com/micronaut-projects/micronaut-starter/pull/1189/commits/6421e0ef0141f3010069946dbeef148617174c62